CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 6.0.0-dev
 
+* Add support to GFM extension for github task lists (aka checkboxes).  These are only active in the gitHubFlavored and gitHubWeb extension sets.
```

Comment:
Casing nit: GitHub.

Also, let's put `gitHubFlavored` and `gitHubWeb` in backticks.